docs/cli/storage/import.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Use an existing S3 bucket or DynamoDB table
+description: Configure the Amplify CLI to use existing S3 bucket or DynamoDB table resources as a storage resource for other Amplify categories. (API, function, and more)
+---
+
+Import an existing S3 bucket or DynamoDB tables into your Amplify project. Get started by running `amplify import storage` command to search for & import an S3 or DynamoDB resource from your account. 
+
+```sh
+amplify import storage
+```
+
+Make sure to run `amplify push` to complete the import process and deploy this backend change to the cloud. 
+
+The `amplify import storage` command will:
+* automatically populate your Amplify Library configuration files (aws-exports.js, amplifyconfiguration.json) with your chosen S3 bucket information 
+* provide your designated S3 bucket or DynamoDB table as a storage mechanism for all storage-dependent categories 
+* enable Lambda functions to access the chosen S3 or DynamoDB resource if you permit it
+
+This feature is particularly useful if you're trying to:
+* enable Amplify categories (such as API and function) to access your existing storage resources;
+* incrementally adopt Amplify for your application stack;
+* independently manage S3 and DynamoDB resources while working with Amplify.
+
+## Import an existing S3 bucket
+
+Select the "Content (Images, audio, video, etc.)" option when you've run `amplify import storage`. In order to successfully import your S3 bucket, your S3 bucket must be hosted in the same region as your Amplify project.
```

Comment:
We dont filter buckets based on region. One can have cross region buckets as those are considered global resources.
```suggestion
Select the "Content (Images, audio, video, etc.)" option when you've run `amplify import storage`.
```

Reply:
Addressed the issue.